src/customizations/volto/components/manage/Blocks/Search/SearchBlockView.jsx

```diff
@@ -100,6 +101,10 @@ const SearchBlockView = (props) => {
     dataListingBodyVariation,
   );
 
+  const [defaultListingBodyData, setDefaultListingBodyData] = React.useState(
+    applyDefaults(searchData, root, data.usePloneRanking),
+  );
```

Comment:
Dato che il metodo `set...` non viene usato, possiamo non usare uno state? Sembra una cosa che può essere semplicemente `const defaultListingBodyData = applyDefaults(searchData, root, data.usePloneRanking);` o qualcosa del genere

Reply:
abbiamo gia una variabile li dentro che si chiama `listingBodyData` che viene inizializzata allo stesso modo di quella nuova che ho introdotto io (`defaultListinBodyData`).
Il problema è che listingBodyData cambia ogni volta che l'utente applica dei filtri e ci si perde il default iniziale. Settandolo nello state del componente invece viene mantenuto inalterato il suo valore iniziale in modo da poter fare i controlli che servono per decidere se scrollare o meno all'inizio dei risultati. 

Ora ho rimosso il metodo `set..` che tanto non serviva